bccsp/schemes/idemixopts.go

```diff
@@ -237,6 +237,9 @@ func (o *IdemixCredentialSignerOpts) IssuerPublicKey() Key {
 }
 
 type NymEIDAuditData struct {
+	// Nym is the EID Nym
+	Nym *math.G1
```

Comment:
maybe we can add a comment to specify that this field is not guaranteed to be correct until verify is called?

Reply:
added a comment to the struct